distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/Worker.java

```diff
@@ -244,48 +238,65 @@ private void registerWorkerAsAliveAndRegisterJobs(
         workPooledCache.start();
     }
 
+    /**
+     * Add in zk paths of new work items, that contains in newWorkPool, but doesn't in currentWorkPool and
+     * remove work items, that contains in currentWorkPool, but doesn't in newWorkPool.
```

Comment:
convert comments to two functions and call them from mergeWorkPoolsForJob

Reply:
:thinking: